compiler/luci/service/src/CircleShapeSignatureInferenceHelper.cpp

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Service/CircleShapeSignatureInferenceHelper.h"
+
+namespace ShapeSignatureInferenceHelper
```

Comment:
This is a very long namespace name and is camel case.
We usually used lowe case, short name.
I don't have any particular perference but this is strange.

Reply:
At first, I was not sure whether I can put these functions to `luci`.
If you don't mind, I can fix as following codes.

```cpp
namespace luci
{

luci::ShapeSignature signature_of_input(const luci::CircleNode *node, uint32_t index);

} // namespace luci
```

Reply:
> At first, I was not sure whether I can put these functions to luci.

codes exist in `luci` folder. I am more curious what may bother you putting in `luci` namespace...